assets/static/js/custom_scripts.js

```diff
@@ -4,34 +4,52 @@
   "use strict";
 
   /* Top-level variables */
-  var buttonData = {
-    win: {
-      text: "Download for Windows",
-      icon: ["fab", "fa-windows"],
-      url: "https://github.com/spyder-ide/spyder/releases/latest/download/Spyder_64bit_full.exe",
-    },
-    mac: {
-      text: "Download for macOS",
-      icon: ["fab", "fa-apple"],
-      url: "https://github.com/spyder-ide/spyder/releases/latest/download/Spyder.dmg",
-    },
-    linux: {
-      text: "Download for Linux (Anaconda)",
-      icon: ["fab", "fa-linux"],
-      url: "https://www.anaconda.com/download/",
-    },
-    other: {
-      text: "Download Spyder",
-      icon: ["fas", "fa-download"],
-      url: "https://github.com/spyder-ide/spyder/releases/latest",
-    },
+  const buttonsData = {
+    win: [
+      {
+        id: "download-windows",
+        text: "Download for Windows",
+        icon: ["fab", "fa-windows"],
+        url: "https://github.com/spyder-ide/spyder/releases/latest/download/Spyder_64bit_full.exe",
+      },
+    ],
+    mac: [
+      {
+        id: "download-mac-m1",
+        text: "Download for macOS (M1)",
```

Comment:
```suggestion
        text: "Download for macOS (M1/M2)",
```

Just in case users get confused and asume the installers don't work for M2 chips (I don't know how common that could be).

Reply:
I thought quite a bit about this issue, but concluded that M1 seems to be more or less accepted as the de-facto short name for Apple Silicon, and while I also considered using ARM as the name, ISTM that average users are likely as or more familiar with M1 (and Intel) than ARM (and x86-64) as the user-facing name of their computer's processor.

In fact, [GitHub's own blog post announcing the Apple Silicon runners](https://github.blog/changelog/2024-01-30-github-actions-introducing-the-new-m1-macos-runner-available-to-open-source/), that started this whole thing to begin with, calls them "M1" and doesn't even mention "ARM", and likewise the previous confusion on this same point when asking GitHub about this stemmed from calling them "ARM64" and not "M1", which would have been more recognizable.

Furthermore, there are some problems with this approach: it much more strongly implies that it _won't_ work on M3+; we'd thus need to remember to update it every year or two when Apple releases a new processor generation, and furthermore it would get ever-longer; even just `M1/M2` is pushing right up near the max width that will fit near certain breakpoints.

In any case, since this PR fixes a critical bug with the current site and is an unambiguous improvement over the current situation, I would strongly advise that we merge this now and save the bikeshedding over exactly what to name this for a followup, if desired.

Reply:
> In fact, [GitHub's own blog post announcing the Apple Silicon runners](https://github.blog/changelog/2024-01-30-github-actions-introducing-the-new-m1-macos-runner-available-to-open-source/), that started this whole thing to begin with, calls them "M1" and doesn't even mention "ARM", and likewise the previous confusion on this same point when asking GitHub about this stemmed from calling them "ARM64" and not "M1", which would have been more recognizable.

Good point.

> Furthermore, there are some problems with this approach: it much more strongly implies that it won't work on M3+; we'd thus need to remember to update it every year or two when Apple releases a new processor generation, and furthermore it would get ever-longer; even just M1/M2 is pushing right up near the max width that will fit near certain breakpoints.

Good point too.

> In any case, since this PR fixes a critical bug with the current site and is an unambiguous improvement over the current situation, I would strongly advise that we merge this now and save the bikeshedding over exactly what to name this for a followup, if desired.

I don't have a strong opinion about this and I agree with your rationale, so merging.